src/python/WMCore/Services/DBS/DBSReader.py

```diff
@@ -17,17 +17,13 @@ def DBSReader(endpoint, **kwargs):
     
     msg = ''
     try:
-        dbs = _getDBS3Reader(endpoint, **kwargs)
+        dbs = DBS3Reader(endpoint, **kwargs)
```

Comment:
Eric,
the code is redundant by itself, nobody calls _getDBS3Reader except
getDBS3Reader on line 20
https://github.com/dmwm/WMCore/pull/6260/files#diff-274504b8b0136ef92367790e745914b5L20

I just cleared it up because on my previous commits you asked explicitly to
move import/from statements in top of the files. I made it and remove
extra callback. But this change is indeed irrelevant for purpose of database
discussion.

On  0, Eric Vaandering <notifications@github.com> wrote:
> > @@ -17,17 +17,13 @@ def DBSReader(endpoint, **kwargs):
> >      
> >      msg = ''
> >      try:
> > -        dbs = _getDBS3Reader(endpoint, **kwargs)
> > +        dbs = DBS3Reader(endpoint, **kwargs)
> 
> This is the set of changes that I am wondering why you need to make it. Granted, I want to get rid of all this with what @jha2 is doing, but I don't see what this has to do with MariaDB.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6260/files#r39900934